database/migrations/2024_12_10_014233_create_participant_state_form_authorization_for_confidential_information.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('participant_state_form_authorization_for_confidential_information', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('participant_id')->constrained();
+            $table->string('consumer_name');
+            $table->boolean('dss_authorized');
+            $table->boolean('fys_authorized');
+            $table->boolean('mhd_authorized');
+            $table->boolean('dfas_authorized');
+            $table->boolean('mmac_authorized');
+            $table->boolean('fsd_authorized');
+            $table->boolean('cd_authorized');
+            $table->boolean('dls_authorized');
+            $table->string('other_authorized_entity')->nullable();
+            $table->string('disclosure_entity_name');
+            $table->string('disclosure_entity_phone');
+            $table->date('disclosure_entity_dob');
+            $table->string('disclosure_entity_ssn')->nullable();
+            $table->string('disclosure_entity_address');
+            $table->string('disclosure_entity_email')->nullable();
+            $table->boolean('attorney_recipient');
```

Comment:
Can we turn these booleans into verbs?  Something like disclosed_to_attorney?
These will also need strings to write in names